### PR TITLE
Update ma.m3u

### LINKED ISSUE #6220: Replace: Medi 1 TV group
### Channel Name

Medi 1 TV Afrique/Arabic/Maghreb

### What happened to the stream?

It is buffering to unplayable point

### Playlist

https://iptv-org.github.io/iptv/countries/ma.m3u 

### Broken Link

_No response_

### Possible Replacement

https://www.medi1tv.ma/fr/live-medi1tv-afrique / https://www.medi1tv.ma/fr/live-medi1tv-arabic / https://www.medi1tv.ma/fr/live-medi1tv-maghreb 

### Notes

I don't know if the m3u8 links can be taken out.  I don't have the tools. Tanks you !

### LINKED ISSUE #6252: Replace: 2m Marroc
### Channel Name

2m Marroc

### What happened to the stream?

It is buffering to unplayable point

### Playlist

https://iptv-org.github.io/iptv/countries/ma.m3u

### Broken Link

https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/2m_monde/hls_video_ts/2m_monde.m3u8

### Possible Replacement

_No response_

### Notes

_No response_

### PR DESCRIPTION
**Pull request by Sphinxroot**
close #6220
close #6252

### CODE REVIEW

**Comment by Dum4G** [comment]
VLC shows me certificate thing again

**Review by dct-infra — COMMENTED** [review]
I guess we can safely remove https from both urls.

**Comment by LaneSh4d0w** [comment]
Works for me.

**Review by dct-infra — DISMISSED** [review]

**Review by Sphinxroot — COMMENTED** [review]
Medi 1 TV Arabic 1080p
http://streaming1.medi1tv.com/live/smil:medi1ar.smil/playlist.m3u8   i missed the change

File: channels/ma.m3u [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="2MMonde.ma" tvg-country="MA" tvg-language="Arabic;French" tvg-logo="https://i.imgur.com/CwwRn4b.png" group-title="General",2M Monde (360p) [Geo-blocked]
```
Comment (Dum4G) [inline channels/ma.m3u:2]: isn't geo-blocked for me. Probably it was a dead link before this PR

File: channels/ma.m3u [hunk 2]
```diff
@@ -25,18 +25,20 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/arriadia/hls_snrt/index.
...
 #EXTINF:-1 tvg-id="M24TV.ma" tvg-country="MA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/J7GdM3L.png" group-title="News",M24 TV (720p) [Not 24/7]
 https://www.m24tv.ma/live/smil:OutStream1.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="MaghrebArabePress.ma" tvg-country="MA" tvg-language="Arabic" tvg-logo="" group-title="",Maghreb Arabe Press (720p) [Not 24/7]
 https://www.maptvnews.ma/live/smil:OutStream1.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Medi1TVAfrique.ma" tvg-country="MA" tvg-language="French" tvg-logo="https://i.imgur.com/MosTwQW.png" group-title="News",Medi 1 TV Afrique (1080p) [Not 24/7]
-https://5f72f3a9b06b7.streamlock.net/live/smil:medi1fr.smil/playlist.m3u8
+http://streaming2.medi1tv.com/live/smil:medi1fr.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Medi1TVArabic.ma" tvg-country="MA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/MosTwQW.png" group-title="News",Medi 1 TV Arabic (1080p) [Not 24/7]
-https://5f72f3a9b06b7.streamlock.net/live/smil:medi1ar.smil/playlist.m3u8
+http://5f72f3a9b06b7.streamlock.net/live/smil:medi1ar.smil/playlist.m3u8
```
Comment (Dum4G) [inline channels/ma.m3u:39]: duplicates Medi 1 TV Maghreb

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/ma.m3u (hunk 2 above)
Comment (dct-infra) [reply channels/ma.m3u:39]: Indeed, we have 2 names for the same content. Both channels exist and according to lyngsat each of these 2 channels covers a specific geo zone.
I suppose It's just a naming thing that depends on the targeted audience. We should keep both urls and rename one of them eventually.

Comment (LaneSh4d0w) [reply channels/ma.m3u:39]: There's one version for the citizens living in the Maghreb diaspora, and one for ISPs, often being Medi1 France.

Comment (Dum4G) [reply channels/ma.m3u:39]: I mean it's literally live/smil:medi1ar.smil/playlist in both cases. New links are probably redirectional to a proper CDN and that's it. See my suggestion below

File: channels/ma.m3u [hunk 3]
```diff
@@ -25,18 +25,20 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/arriadia/hls_snrt/index.
...
 #EXTINF:-1 tvg-id="Medi1TVMaghreb.ma" tvg-country="MA" tvg-language="French;Arabic" tvg-logo="https://i.imgur.com/MosTwQW.png" group-title="News",Medi 1 TV Maghreb (1080p) [Not 24/7]
-https://5f72f3a9b06b7.streamlock.net/live/smil:medi1tv.smil/playlist.m3u8
+http://streaming1.medi1tv.com/live/smil:medi1ar.smil/playlist.m3u8
```
Comment (Dum4G) [inline channels/ma.m3u:41]:
```suggestion
http://streaming1.medi1tv.com/live/smil:medi1tv.smil/playlist.m3u8
```

**Review by Dum4G — APPROVED** [review]

**Review by dct-infra — APPROVED** [review]

**Pull request merged**